multicast/env.py

```
@@ -248,10 +248,10 @@
 	try:
 		ttl_num = int(ttl)
 		return 1 <= ttl_num <= 126
-	except (ValueError, TypeError) as err:
+	except (ValueError, TypeError) as _cause:
 		raise ValueError(
 			f"Invalid TTL value: {ttl}. Must be a positive integer below 127."
-		) from err
+		) from _cause
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)